### PR TITLE
second task done

### PR DESCRIPTION
**Pull request by kkkirill**
#### Distributed requests processing on 2 servers:
* Server 0:
	* GET /form
* Server 1:
	* GET /authh/login
	* GET /authh/logout
	* POST /charge
### How to Verify 
See README.md

### CODE REVIEW

File: HTTPlayground/settings.py [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
-URL = 'localhost'
+SERVERS_INFO = {
+    'URL0': '127.0.0.1', 'PORT0': 8000,
+    'URL1': '0.0.0.0', 'PORT1': 8080  # 127.255.255.254
+}
 
-if len(argv) > 2:
-    PORT = int(argv[2])
-    URL = argv[1]
+IS_MAIN_SERVER = not int(argv[1])
```
Comment (andrey-bulygin) [inline HTTPlayground/settings.py]: minor: this doesn't look explicit at all. Always try to think of more self-explanatory implementation options.

File: HTTPlayground/settings.py [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+
+PORT = SERVERS_INFO.get(f'PORT{argv[1]}', SERVERS_INFO['PORT0'])
+URL = SERVERS_INFO.get(f'URL{argv[1]}', SERVERS_INFO['URL0'])
+
+ACCESSORY_PORT = SERVERS_INFO.get(f"PORT{1 - int(argv[1])}", "PORT1")
```
Comment (andrey-bulygin) [inline HTTPlayground/settings.py]: I think you might want `ACCESSORY_*` creds to be `None` for the second server, not the ones of the current server itself.

File: HTTPlayground/settings.py [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+    'URL0': '127.0.0.1', 'PORT0': 8000,
+    'URL1': '0.0.0.0', 'PORT1': 8080  # 127.255.255.254
+}
 
-if len(argv) > 2:
-    PORT = int(argv[2])
-    URL = argv[1]
+IS_MAIN_SERVER = not int(argv[1])
+
+PORT = SERVERS_INFO.get(f'PORT{argv[1]}', SERVERS_INFO['PORT0'])
```
Comment (andrey-bulygin) [inline HTTPlayground/settings.py]: I wouldn't fall back to any default value here. Usually one wants to be specific on what they want to run, so they do need to supply an explicit value.

File: HTTPlayground/settings.py (hunk 1 above)
Comment (andrey-bulygin) [reply HTTPlayground/settings.py]: advice: define smth like `main|accessory` values for the argv, not ints. They would speak more to the user (who may be a different person in theory)

File: templates/login.html [hunk 4]
```diff
@@ -1,13 +0,0 @@
-<!DOCTYPE html>
```
Comment (andrey-bulygin) [inline templates/login.html:1]: Are you sure to have removed those templates? They seem to be missing in this task branch. Sure to have checked that everything works properly indeed?

**Review by andrey-bulygin — APPROVED** [review]
Looks pretty good as of now.
Mind all new comments though, and preferrably give your vision in further comments.

File: templates/login.html (hunk 4 above)
Comment (kkkirill) [reply templates/login.html:1]: Everything is working correctly.


> Are you sure to have removed those templates? They seem to be missing in this task branch. Sure to have checked that everything works properly indeed?

File: HTTPlayground/settings.py (hunk 3 above)
Comment (kkkirill) [reply HTTPlayground/settings.py]: Ok, will remove default value.

> I wouldn't fall back to any default value here. Usually one wants to be specific on what they want to run, so they do need to supply an explicit value.

File: HTTPlayground/settings.py (hunk 2 above)
Comment (kkkirill) [reply HTTPlayground/settings.py]: In the response template of the second server the url of the main server is used (in the "Home" button).

> I think you might want `ACCESSORY_*` creds to be `None` for the second server, not the ones of the current server itself.

File: templates/login.html (hunk 4 above)
Comment (andrey-bulygin) [reply templates/login.html:1]: 👍

File: HTTPlayground/settings.py (hunk 2 above)
Comment (andrey-bulygin) [reply HTTPlayground/settings.py]: OK.

**Pull request merged**